bin/fileloadhelpers/FileLoadHelpers.cpp

```diff
@@ -0,0 +1,140 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "stdafx.h"
+#include "Codex/Utf8Codex.h"
+
+HRESULT FileLoadHelpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents, bool* isUtf8Out, LPCWSTR* contentsRawOut, UINT* lengthBytesOut, bool printFileOpenError)
+{
+    HRESULT hr = S_OK;
+    LPCWSTR contentsRaw = nullptr;
+    byte * pRawBytes = nullptr;
+    UINT lengthBytes = 0;
+    bool isUtf8 = false;
+    contents = nullptr;
+    FILE * file;
+
+    //
+    // Open the file as a binary file to prevent CRT from handling encoding, line-break conversions,
+    // etc.
+    //
+    if (fopen_s(&file, filename, "rb") != 0)
+    {
+        if (printFileOpenError)
+        {
+#ifdef _WIN32
+            DWORD lastError = GetLastError();
+            char16 wszBuff[512];
+            fprintf(stderr, "Error in opening file '%s' ", filename);
+            wszBuff[0] = 0;
+            if (FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM,
```

Comment:
@obastemur  This code refactored from the ch\helpers.cpp. Nothing new is added here than it was before.